package.json

```diff
@@ -16,54 +16,49 @@
     "test": "test"
   },
   "scripts": {
-    "build": "gulp build",
-    "watch": "gulp build watch",
-    "deploy": "npm run lint && npm run test && scripts/deploy.sh",
-    "lint": "gulp jshint",
-    "test": "gulp t"
+    "build": "tsc && browserify src/cp.js -d -s cp | exorcist compass.js.map > compass.js",
```

Comment:
@yhoonkim  To understand build systems, look into what these commands are for.  

`tsc` = compile TS
`browserify` = combining files
`exorcist` = source map.